neurom/io/check.py

```diff
@@ -79,7 +77,7 @@ def has_increasing_ids(raw_data):
     with their predecessor)
     '''
     ids = raw_data.get_col(COLS.ID)
-    steps = [int(j) for (i, j) in zip(ids, ids[1:]) if i >= j]
+    steps = ids[np.where(np.diff(ids) <= 0)[0] + 1].astype(int)
```

Comment:
Yes, my bad on this. They should be converted into ints or be ints from the beginning. I've never used the mixed arrays, thus I've no idea of the overhead.